docs/Style-Guide.md

```diff
@@ -0,0 +1,88 @@
+# Style Guide
+
+This project follows four style + structure considerations:
+
+1. JavaScript code must conform to [`standard`](#js-standard).
+2. All Vue.js-related files and code should follow the [_Priority A_ and _Priority B_ Rules](#vuejs-style-guide) in the Vue.js Style Guide.
+3. You must understand and follow [Group Income's data organization model](#group-income-data-model-rules).
+4. [SBP](#SBP). (Soon.)
+
+Please read all the sections below before writing a single line of code.
+
+## JS Standard
+
+While our build system should automatically detect most deviations from [JS Standard](https://github.com/standard/standard), it might not catch all (especially in `.vue` files).
+
+If you notice any files not properly linted by `standard`, this means there's a bug, and you're welcome to help fix it!
+
+_It is still on you to ensure your code conforms to the `standard` spec, whether or not the linter catches everything._
+
+## Vue.js Style Guide
+
+Since this is a Vue.js project, any pull requests **must** follow *Priority A* rules mentioned in the [Vue.js Style Guide](https://vuejs.org/v2/style-guide/), and *should* follow the *Priority B* rules. Please take the time to read at least those two sections.
+
```

Comment:
I'm not sure this is the best place for this, but maybe we could capture here what we agreed on with @willstanley about css styling as point 3. Something along these lines:

## CSS structuring guide

For styling, we use [Bulma v 0.4.2. ](https://versions.bulma.io/0.4.4/documentation/elements/box/)
- Everything that can be solved by using Bulma’s classes should be solved with them
- Theme overwrites that affect the whole application - for elements that could be found at the [Bulma docs](https://versions.bulma.io/0.4.4/documentation/elements/box/) - should go to `/frontend/simple/sass` and be written in SCSS
- Component specific styles should go to the component’s `<style>` tag, be written in SCSS, and be scoped to the component. Try to write as little of this as possible.

Reply:
@hubudibu great suggestion! Will add this section!